integration-tests/api/__tests__/store/products/ff-product-categories.ts

```diff
@@ -219,6 +220,25 @@ describe("/store/products", () => {
       )
     })
 
+    it("returns only active and public products with include_category_children when categories are expanded", async () => {
```

Comment:
suggestion: I forgot to add this, but it would be good to have a test that returns actual categories. 

Let me know if you want me to add it here.

Reply:
on it

Reply:
I ve updated the existing test as it make sense. wdyt?

https://github.com/medusajs/medusa/pull/4084/commits/eb02393b7140479b531111ee109bea210b53856b

Reply:
Legend! 💪🏻